docs/mobile/exercises/README.md

```diff
@@ -0,0 +1,294 @@
+# Mobile exercises
+
+Writing a DataCamp for Mobile is quite different than writing a DataCamp
+[desktop course](../../courses/readme.md). The upside is that you can reach a
+much broader audience because the convenience is higher and the barrier to entry
+is lower. The challenge is that there are many more constraints to work
+within. Primary among these are the restricted screen real estate and the lack
+of an interactive interpreter. These constraints require you to think carefully
+about the way you introduce concepts and the examples that you use.
+
+# Basic exercise structure
+
+There are roughly 5 exercise types available to you as an author on DataCamp
+Mobile, but they all share a similar structure. In general (with a few
+exceptions), you can think of an exercise as a short code vignette comprising
+various blocks in the following structure:
+
+```yaml
+# print()
+- key: c11dbdf1-49c9-408a-a1e4-885431af2196
+  context: "`print()` outputs to the screen. Whatever you put between the parentheses will be output! E.g. `print(5)` will output `5`!"
+  question: "" # this is the default
+  code: "print(42)"
+  output: "42"
+  distractor_output:
+    - option: "4"
+      feedback: "`print()` will output all of whatever is between the parentheses, not just the first part."
+    - option: "2"
+      feedback: "`print()` will output all of whatever is between the parentheses, not just the last part."
+```
+
+Let's dig into each of the blocks...
```

Comment:
Before you step through each yaml element (and preferably before the example code), I'd give a summary of what the constituent elements  of an exercise are.  Something like:

Exercises consist of three or four compulsory elements (depending upon exercise type), and some optional elements. The compulsory elements are `context`, which contains the explanatory text at the start of the exercise, `code` which contains the sample code, `output` which contains the correct output from running that code, and `distractor_output` which contains feedback in the event of an incorrect answer.